crates/viewer/re_view/src/results_ext.rs

```diff
@@ -94,15 +94,15 @@ impl HybridLatestAtResults<'_> {
     pub fn get_instance<C: re_types_core::Component>(
         &self,
         index: usize,
-        component_descr: &ComponentDescriptor,
+        component_descr: &ComponentDescriptor, // TODO(#10460): Take `ComponentIdentifier` instead of descriptor.
```

Comment:
Btw, we should probably do `blueprint_helpers.rs` (reset, ...) as part of #10460 too.